lunes_cms/cms/models/feedback.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
```

Comment:
```suggestion

MATTER_CHOICES = (
    ("GENERAL", _("general")),
    ("DISCIPLINE", _("discipline")),
    ("TRAINING_SET", _("training set")),
    ("DOCUMENT", _("document")),
)


```
I'd move the matter choices outside of the model definition.
Also, the second part of the tuple is usually the "readable" part, so it makes sense to translate them.